source/_integrations/elevenlabs.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: ElevenLabs text-to-speech
+description: Instructions on how to setup ElevenLabs text-to-speech with Home Assistant.
+ha_category:
+  - Text-to-speech
+ha_release: 2024.8
+ha_iot_class: Cloud Push
+ha_domain: elevenlabs
+ha_platforms:
+  - tts
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The **ElevenLabs** text-to-speech {% term integrations %} adds support for [ElevenLabs](https://elevenlabs.io/) as text-to-speech to read a text with natural sounding voices.
+## Prerequisites
+
+- You need an ElevenLabs account to use this integration. A free account suffices for basic usage. 
+- For custom voices or more quota you need a subscription.
+- You need your API key from the ElevenLabs website. 
+
+{% include integrations/config_flow.md %}
+
+For an overview of which languages can be used, check the [complete list of supported languages](https://elevenlabs.io/languages).
+
+For a description of which voices are available for use, check your VoiceLab voices.
+
+To see the available models and their benefits, check the [models documentation](https://elevenlabs.io/docs/speech-synthesis/models).
+
+## Action speak
+
+The `tts.speak` action is the modern way to use TTS. Add the `speak` action, select the entity for your ElevenLabs TTS (it's named ElevenLabs), select the media player entity or group to send the TTS audio to, and enter the message to speak.
```

Comment:
The entity isn't always called eleven labs right?

Reply:
We're naming it based on the config entry title and the config entry title is ElevenLabs
That should result in the Entity's name being ElevenLabs, shouldn't it?

I agree that we can discuss this though as a more meaningful name would be nice, but I'd move that into a separate discussion in core

Reply:
Should we add `by default`?

Reply:
```suggestion
The `tts.speak` action is the modern way to use TTS. Add the `speak` action, select the entity for your ElevenLabs TTS (it's named ElevenLabs by default), select the media player entity or group to send the TTS audio to, and enter the message to speak.
```